packages/plugin-ext/src/plugin/types-impl.ts

```diff
@@ -1457,7 +1457,7 @@ export class ProcessExecution {
     }
 
     public computeId(): string {
-        const hash = crypto.createHash('md5');
+        const hash = crypto.createHash('sha256');
```

Comment:
Not sure about backward computability consequences of changing this file. Do we store them somewhere and should read again or it is pure runtime thing?

Reply:
As I see, it's used at runtime only.
BTW I've found that it has been reworked in VS Code to generate IDs based on the command line instead of hashes.
https://github.com/microsoft/vscode/commit/36d9d043b94f610aaf2683604f1caa8c94bdec5d
We need to update it in Theia correspondingly. I'll take a look at whether it can be done within this PR.

Reply:
I've changed the IDs generation according to the current VS Code state. Now, it doesn't require `crypto` Node module.